cli/spec.py

```diff
@@ -0,0 +1,244 @@
+import clg
+import ConfigParser
+import os
+import sys
+import yaml
+import yamlordereddictloader
+
+from cli import exceptions
+from cli import utils
+
+SPEC_EXTENSION = '.spec'
+LOOKUP_DICT = {
+    '__DEFAULT__': "default"
+}
+
+TRIM_PARAMS = ['default', 'required', 'requires_only']
+
+
+def IniFileType(value):
+    """
+    The custom type for clg spec
+    :param value: the argument value.
+    :return: dict based on a provided ini file.
+    """
+    _config = ConfigParser.ConfigParser()
+    _config.read(value)
+
+    d = dict(_config._sections)
```

Comment:
Please use more meaningful names for variables, and try to eliminate call directly to non-public variables, especially when you have methods for that.

Make the code more readability (for example):
res_dict = {}
for section in  config.sections():
    res_dict[section] = {} 
    for option,value in config.items(section):
        res_dict[section][option] = value
...
return res_dict